doc/openocd.texi

```diff
@@ -10738,6 +10738,23 @@ For details on this trigger type, see the RISC-V Debug Specification.
 Clear the type 5 trigger that was set using @command{riscv etrigger set}.
 @end deffn
 
```

Comment:
I wonder if we should note in the doc that the triggers are an optional feature - making it clearer to the user that  the commands will not generally work on any RISC-V target.

For instance:
`These commands let you set those triggers directly.` ---> `The triggers are an optional feature. The following commands allow to set certain trigger types, provided they are supported by the given RISC-V target.`
(or similar wording)

Reply:
Good point. Added something like that.